Neuro/Tasks/TasksHandler.cs

```diff
@@ -20,43 +20,42 @@ public class TasksHandler : MonoBehaviour
     [HideFromIl2Cpp]
     public void UpdatePathToTask(PlayerTask task = null)
     {
-        if (!task) task = PlayerControl.LocalPlayer.myTasks.At(0);
+        bool isImpostor = NeuroPlugin.Instance.Impostor.isImpostor;
+        if (!task)
+        {
+            // as impostor, among us adds a fake task to the front of the list with no position
+            // therefore skip it as it cannot be completed
+            if (isImpostor)
+                task = PlayerControl.LocalPlayer.myTasks.At(1);
```

Comment:
This task is of type ImportantPlayerTask, please skip it by its type and not by its index